dipy/io/dpy.py

```diff
@@ -99,8 +113,15 @@ def write_track(self, track):
         self.offsets.resize(self.offsets.shape[0] + 1, axis=0)
         self.offsets[-1] = self.curr_pos
 
-    def write_tracks(self, tracks):
+    def write_tracks(self, tracks: Streamlines) -> None:
```

Comment:
same comment as above:

Can you follow numpy documentation standard : https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
okay